### ISSUE TITLE
How  to query HttpRequestData .net 5  (Query) gone!

### ISSUE THREAD

**Issue by developer9969** [Azure/azure-functions-dotnet-worker#502]
Hi
This is not an issue at all, but rather a question on how to do something 

In the "old" .net core we could do  ]

    ` var myfield =req.Query["MyField"]`

How do you do it using HttpRequestData now?

Hopefully it will be a quick straight answer 

thanks

**Comment by Elip-tdlr** [comment]
I believe the best way to do this us using  [Microsoft.AspNetCore.WebUtilities.ParseQuery](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.queryhelpers.parsequery?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery);k(DevLang-csharp)%26rd%3Dtrue&view=aspnetcore-5.0)

I create an extension method like this to facilitate getting query params from `HttpRequestData`

`public static Dictionary<string, StringValues> GetQueryParams(this HttpRequestData req) => QueryHelpers.ParseQuery(req.Url.Query);`

**Comment by developer9969** [comment]
Hi that will do but seems a bit of a step back as before it was in built in

**Comment by ijagberg** [comment]
Also interested in this, feels like accessing query parameters should be a built-in feature.

**Comment by anthonychu** [comment]
Makes sense. We should expose this as a first class property.

**Comment by developer9969** [comment]
Cool thanks @anthonychu  , hopefully we will not have to wait long  :)

**Comment by calloncampbell** [comment]
Any update on this with .NET 6 (isolated)?

**Comment by kshyju** [comment]
The fix is available in [worker package version 1.13.0](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker/1.13.0)

https://github.com/Azure/azure-functions-dotnet-worker/releases/tag/1.13.0

#### Sample usage

```csharp
public HttpResponseData Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequestData req)
{
    string? fooValue = req.Query["foo"];
    .....
}
```
